test/src/tests-kit-blocking.ts

```diff
@@ -616,6 +617,24 @@ describe('kit blocking', () => {
             })
         });
 
+        it('integration test - should not throw an error when unplanned user attributes are allowed and block.ua = true', function(done) {
+            window.mParticle.config.kitConfigs.push(forwarderDefaultConfiguration('MockForwarder'));
+            window.mParticle.init(apiKey, window.mParticle.config);
+            
+            //save old data points for reset later
+            const oldDataPoints = dataPlan.dtpn.vers.version_document.data_points;
+            dataPlan.dtpn.vers.version_document.data_points = [];
```

Comment:
Add a comment specifying that when "Allow planned user attributes" is set, that data_points returns an empty array.